core/src/gl/texture.cpp

```diff
@@ -8,6 +8,14 @@
 #include "tangram.h"
 #include "log.h"
 
+// Only JPEG, PNG and GIF
+#define STBI_NO_BMP
+#define STBI_NO_PSD
+#define STBI_NO_TGA
```

Comment:
Should we though? I agree that for compatibility with JS, we can. But if tangram-es is being used by a game developer exporting textures from photoshop or other softwares, PSD, BMP, TGA are some of the textures artists, game developers, students use all the time.